README.md

```diff
@@ -18,7 +18,12 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.deploygate:sdk:<latest version>'
+    // use full implementation for debug builds
+    debugImplementation 'com.deploygate:sdk:<latest version>'
+    
+    // use mocked implementation for release builds
```

Comment:
```suggestion
    // Use no-op implementation for variants you would like to disable DeployGate SDK.
```

Reply:
Sound good to me.
The scene of need to disable DeployGate SDK maybe not only release build 💡